browser/util.js

```diff
@@ -15,56 +15,23 @@ import * as config from './config.js';
  *     frameworks have loaded.
  */
 export function whenFrameworksReady(callback) {
-  debug('whenFrameworksReady');
-  var done = function() {
-    debug('whenFrameworksReady done');
+  // webcomponents-loader.js will eventually fire WebComponentsReady.
+  // If WebComponents is not defined and we cannot find the script,
+  // invoke the callback.
+  if (!window.WebComponents &&
+    !document.querySelector('script[src*="webcomponents-loader.js"]')) {
```

Comment:
When is whenFrameworksReady called? Can we wait for all scripts to finish loading first, so that we don't have to look for script tags with special srcs?